content/7-webapp-features/1-fetch-data/vue3/useFetchUsers.js

```diff
@@ -0,0 +1,24 @@
+import { ref } from 'vue';
+
+export default function useFetchUsers() {
+	const data = ref(null);
+	const error = ref(null);
```

Comment:
No need to precise `null` value. `ref()` is like `ref(undefined)`.
And it would match React & Svelte examples :)